README.md

```diff
@@ -102,29 +102,29 @@ _Last updated by forest2001 on 2025-04-01_
 1. Must be on Discord in Blufor voice channels.
 2. Role: Protect civilians and enforce law.
 3. Use non-lethal force first when possible.
-4. Police Chief must be active on Discord; staff may replace.
-5. Observe rank hierarchy. Staff > Chief.
+4. Police Chief must be active on Discord; staff may replace at their own discretion.
+5. Observe rank hierarchy. Staff > Chief > Higher Ranking Officers.
 6. Must explain citations for them to be valid.
-7. No compensation requests.
-8. Cannot enter the South unless in joint ops or war.
+7. No compensation requests unless a rule was broken.
+8. Cannot enter the South unless in joint ops or war. (without)
 
 ### Kill/Arrest
 
 1. Never kill unarmed Civilians. VDM attempts = armed.
 2. Must carry a stun weapon.
 3. Must give reason for arrests.
-4. Civs in armed police vehicles = KOS; otherwise arrest.
+4. Civs in armed and heavily armoured police vehicles = KOS; otherwise arrest. (anything resistant to rifle is heavily armoured)
```

Comment:
Maybe add an ammo type here instead of "rifle" as 7.62x51mm will go through plenty of vehicles that 5.56 will not.